tests/acceptance/features/webUIResharingToRoot/reshareUsers.feature

```diff
@@ -136,8 +136,8 @@ Feature: Resharing shared files with different permissions
       | shared-entry-name | received-entry-name |
       | simple-folder     | simple-folder (2)   |
       | lorem.txt         | lorem (2).txt       |
-      | simple-folder     | simple-folder (2)   |
-      | lorem.txt         | lorem (2).txt       |
+
+
 
 
```

Comment:
the gap is big. But not keeping this gap will create a lot of changes in the expected failure file.

Reply:
Yes, exactly, I'm trying not to change any line numbers right now.